travis/run-small-python-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -x
+# Set err=1 if any commands exit with non-zero status as described in
+# https://stackoverflow.com/a/42219754
+err=0
+trap 'err=1' ERR
+
+pytest --cov=mlflow --verbose --ignore=tests/h2o --ignore=tests/keras \
```

Comment:
Do we need to duplicate these `--ignore` calls? Couldn't we just mark these as large?

Reply:
Yep true - unfortunately there doesn't seem to be a great way to mark the specific test files/modules as large & it's a little heavyweight to put @pytest.mark.large before each large test, but it is helpful to make the usage of @pytest.mark.large consistent.